src/app/views/app-sections/StatusMessages.tsx

```diff
@@ -4,7 +4,8 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+  isMultiline={false} onDismiss={actions.clearQueryStatus}
+  dismissButtonAriaLabel='Close' aria-live={'assertive'}>
```

Comment:
You can improve readability by arranging the props as shown below.
```
<MessageBar
        messageBarType={}
        isMultiline={}
        onDismiss={}
>
```

Reply:
Oh yeah, thank you for this